### PR TITLE
add how to build, use, test and deploy to README.md

### PR DESCRIPTION
**Pull request by sns-seb**

### CODE REVIEW

**Comment by sns-seb** [comment]
@simonbrandhof do you mind testing the instructions for Mac for me, plz?

I obviously couldn't test them and forged them from when I understood from indirect discussions with @michal-duda-sonarsource on that topic

**Comment by sns-seb** [comment]
@simonbrandhof I update the README according to our discussion. Please review again

File: DEVELOPER.md [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+```
+
+If SQ is running on another port, you must specify it in your project's `sonar-project.properties` with `sonar.host.url=http://localhost:9010` and execute the same `docker run` command.
+
+### With SonarQube running in Docker
+
+First create a network and boot SonarQube:
+
+```
+docker network create "scanner-sq-network"
```
Comment (pierre-guillot-sonarsource) [inline DEVELOPER.md]: why do we need to create a network ? the scanner can use `host.docker.internal`. Beside, i don't see uses cases where sq is running on the same docker host as the scanner.

Comment (pierre-guillot-sonarsource) [reply DEVELOPER.md]: Or is this readme for sq developers ?

Comment (sns-seb) [reply DEVELOPER.md]: `host.docker.internal` works only on mac (maybe on windows with Docker for Windows, but this has not been confirmed by @philippe-perrin-sonarsource )

> Or is this readme for sq developers ?

yes

**Pull request merged**